model/config.go

```diff
@@ -1091,6 +1091,7 @@ type SqlSettings struct {
 	DataSourceSearchReplicas    []string `access:"environment,write_restrictable,cloud_restrictable"`
 	MaxIdleConns                *int     `access:"environment,write_restrictable,cloud_restrictable"`
 	ConnMaxLifetimeMilliseconds *int     `access:"environment,write_restrictable,cloud_restrictable"`
+	ConnMaxIdleTimeMilliseconds *int     `access:"environment,write_restrictable,cloud_restrictable"`
```

Comment:
I understand we want to be consistent with the other setting but I am curious to know if the millisecond precision makes any sense. Could there be reasons to go below 1 second?

Reply:
Yeah I agree. It's mainly legacy. I doubt anyone would know why specifically millisecond precision was chosen. :P